src/porepy/numerics/fv/mpfa.py

```diff
@@ -463,6 +463,15 @@ def _local_discr(self, g, k, bnd, eta=None, inverter="numba", apertures=None):
         # discretization.
         subcell_topology = fvutils.SubcellTopology(g)
 
+        # Below, the boundary conditions should be defined on the subfaces.
```

Comment:
This is a change in the functionality of mpfa, is it not? I am okay with it, but documentation is necessary.

Reply:
It is not a change in the functionality, it is just rewriting the code. We now have bnd.is_neu everywhere instead of bnd.is_neu[subcell_topology.fno_unique].

Reply:
But can you also specify bcs per halfface?

Reply:
yes